src/containers/NewPostForm/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import ReceiverForm from "../../components/ReceiverForm";
+import DonatorForm from "../../components/DonatorForm";
+import { Container, ToggleButton, ButtonGroup, Row } from "react-bootstrap";
+import Logo from "../../logo.png";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+
+export default function Form() {
+  const [radioValue, setRadioValue] = useState("1");
+
+  const { t } = useTranslation();
+
+  const radios = [
+    { name: t("formsection.Receive"), value: "1" },
```

Comment:
You can also give more meaningful values here rather than 1 and 2, to make it clearer and more readable

Reply:
PTAL